### ISSUE TITLE
attacher logs error Failed to list *v1beta1.CSINode: csinodes.storage.k8s.io is forbidden

### ISSUE THREAD

**Issue by gandhisanjayv** [IBM/ibm-spectrum-scale-csi#145]
**Describe the bug**
E0320 19:30:25.432265       1 reflector.go:156] k8s.io/client-go/informers/factory.go:135: Failed to list *v1beta1.CSINode: csinodes.storage.k8s.io is forbidden: User "system:serviceaccount:ibm-spectrum-scale-csi-driver:ibm-spectrum-scale-csi-attacher" cannot list resource "csinodes" in API group "storage.k8s.io" at the cluster scope\
**To Reproduce**
1. ...

**Expected behavior**

**Environment**
Please run the following an paste your output here:
``` bash
# Developement
operator-sdk version 
go version

# Deployment
kubectl version
rpm -qa | grep gpfs
```
**Screenshots**
```
# kubectl get pods   -n ibm-spectrum-scale-csi-driver
NAME                                               READY   STATUS    RESTARTS   AGE
ibm-spectrum-scale-csi-7cfqq                       2/2     Running   0          19h
ibm-spectrum-scale-csi-attacher-0                  1/1     Running   0          19h
ibm-spectrum-scale-csi-b48rp                       2/2     Running   0          19h
ibm-spectrum-scale-csi-h4mvb                       2/2     Running   0          19h
ibm-spectrum-scale-csi-hfj98                       2/2     Running   0          19h
ibm-spectrum-scale-csi-kf6tx                       2/2     Running   0          19h
ibm-spectrum-scale-csi-operator-5f5f549754-xxf5m   2/2     Running   0          6d23h
ibm-spectrum-scale-csi-provisioner-0               1/1     Running   0          19h
ibm-spectrum-scale-csi-qchrt                       2/2     Running   0          19h
ibm-spectrum-scale-csi-v5pw6                       2/2     Running   0          19h
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 12:42:40
# kubectl describe  pods  ibm-spectrum-scale-csi-attacher-0  -n ibm-spectrum-scale-csi-driver
Name:         ibm-spectrum-scale-csi-attacher-0
Namespace:    ibm-spectrum-scale-csi-driver
Priority:     0
Node:         archieprt09/9.11.139.29
Start Time:   Thu, 19 Mar 2020 17:15:38 -0700
Labels:       app=ibm-spectrum-scale-csi-attacher
              app.kubernetes.io/instance=ibm-spectrum-scale-csi-operator
              app.kubernetes.io/managed-by=ibm-spectrum-scale-csi-operator
              app.kubernetes.io/name=ibm-spectrum-scale-csi-operator
              controller-revision-hash=ibm-spectrum-scale-csi-attacher-656b89f997
              product=ibm-spectrum-scale-csi
              release=ibm-spectrum-scale-csi-operator
              statefulset.kubernetes.io/pod-name=ibm-spectrum-scale-csi-attacher-0
Annotations:  cni.projectcalico.org/podIP: 10.244.170.91/32
              productID: ibm-spectrum-scale-csi-operator
              productName: IBM Spectrum Scale CSI Operator
              productVersion: 1.1.0
Status:       Running
IP:           10.244.170.91
IPs:
  IP:           10.244.170.91
Controlled By:  StatefulSet/ibm-spectrum-scale-csi-attacher
Containers:
  ibm-spectrum-scale-csi-attacher:
    Container ID:  docker://14575a21b440b22016c1a72dcf260ca5057d4e036e41561a0b61d7ea355afeac
    Image:         quay.io/k8scsi/csi-attacher:v2.1.1
    Image ID:      docker-pullable://quay.io/k8scsi/csi-attacher@sha256:4bfcf54fde2be69ccf122bc19dd10389013292b49e7d8a57d1eaa013c3672a91
    Port:          <none>
    Host Port:     <none>
    Args:
      --v=5
      --csi-address=$(ADDRESS)
      --resync=10m
      --timeout=2m
    State:          Running
      Started:      Thu, 19 Mar 2020 17:15:45 -0700
    Ready:          True
    Restart Count:  0
    Environment:
      ADDRESS:  /var/lib/kubelet/plugins/ibm-spectrum-scale-csi/csi.sock
    Mounts:
      /var/lib/kubelet/plugins/ibm-spectrum-scale-csi from socket-dir (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from ibm-spectrum-scale-csi-attacher-token-px2ts (ro)
Conditions:
  Type              Status
  Initialized       True
  Ready             True
  ContainersReady   True
  PodScheduled      True
Volumes:
  socket-dir:
    Type:          HostPath (bare host directory volume)
    Path:          /var/lib/kubelet/plugins/ibm-spectrum-scale-csi
    HostPathType:  DirectoryOrCreate
  ibm-spectrum-scale-csi-attacher-token-px2ts:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  ibm-spectrum-scale-csi-attacher-token-px2ts
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:          <none>
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 12:42:52
#
```

**Additional context**

**Comment by gandhisanjayv** [comment]
```
ubectl get serviceaccounts ibm-spectrum-scale-csi-attacher  -n ibm-spectrum-scale-csi-driver -o yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  creationTimestamp: "2020-03-20T00:14:54Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-attacher
  namespace: ibm-spectrum-scale-csi-driver
  resourceVersion: "5387204"
  selfLink: /api/v1/namespaces/ibm-spectrum-scale-csi-driver/serviceaccounts/ibm-spectrum-scale-csi-attacher
  uid: 466b86fa-b54b-4b99-aa5d-cf5dd5610748
secrets:
- name: ibm-spectrum-scale-csi-attacher-token-px2ts
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 12:46:51
```

**Comment by gandhisanjayv** [comment]
```
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 12:59:13
# kubectl get clusterrolebindings  ibm-spectrum-scale-csi-provisioner
NAME                                 AGE
ibm-spectrum-scale-csi-provisioner   19h
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 12:59:28
# kubectl get clusterrolebindings  ibm-spectrum-scale-csi-provisioner -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  creationTimestamp: "2020-03-20T00:15:22Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-provisioner
  resourceVersion: "5387295"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/ibm-spectrum-scale-csi-provisioner
  uid: 581ae30f-c19a-42c1-9b88-85b7f9d27f25
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: ibm-spectrum-scale-csi-provisioner
subjects:
- kind: ServiceAccount
  name: ibm-spectrum-scale-csi-provisioner
  namespace: ibm-spectrum-scale-csi-driver
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 12:59:34
# kubectl get clusterroles ibm-spectrum-scale-csi-provisioner -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: "2020-03-20T00:14:46Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-provisioner
  resourceVersion: "5387177"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterroles/ibm-spectrum-scale-csi-provisioner
  uid: a9f0c483-4f67-4a24-b351-84f9e7f0602a
rules:
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - get
  - list
- apiGroups:
  - ""
  resources:
  - persistentvolumes
  verbs:
  - get
  - list
  - watch
  - create
  - delete
- apiGroups:
  - ""
  resources:
  - persistentvolumeclaims
  verbs:
  - get
  - list
  - watch
  - update
- apiGroups:
  - storage.k8s.io
  resources:
  - storageclasses
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - storage.k8s.io
  resources:
  - csinodes
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - ""
  resources:
  - events
  verbs:
  - list
  - watch
  - create
  - update
  - patch
```

**Comment by gandhisanjayv** [comment]
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  creationTimestamp: "2020-03-20T00:15:14Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-attacher
  resourceVersion: "5387269"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/ibm-spectrum-scale-csi-attacher
  uid: cc5620d6-aefa-463f-93eb-c2400303f365
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: ibm-spectrum-scale-csi-attacher
subjects:
- kind: ServiceAccount
  name: ibm-spectrum-scale-csi-attacher
  namespace: ibm-spectrum-scale-csi-driver
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  creationTimestamp: "2020-03-20T00:15:18Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-node
  resourceVersion: "5387282"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/ibm-spectrum-scale-csi-node
  uid: 32a24ddb-beae-4cc7-a524-d800ca7c9154
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: ibm-spectrum-scale-csi-node
subjects:
- kind: ServiceAccount
  name: ibm-spectrum-scale-csi-node
  namespace: ibm-spectrum-scale-csi-driver
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  creationTimestamp: "2020-03-13T20:00:52Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-operator
  resourceVersion: "3820034"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/ibm-spectrum-scale-csi-operator
  uid: e7680a9b-6289-41ac-a572-dc7325fe8c63
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: ibm-spectrum-scale-csi-operator
subjects:
- kind: ServiceAccount
  name: ibm-spectrum-scale-csi-operator
  namespace: ibm-spectrum-scale-csi-driver
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  creationTimestamp: "2020-03-20T00:15:22Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-provisioner
  resourceVersion: "5387295"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/ibm-spectrum-scale-csi-provisioner
  uid: 581ae30f-c19a-42c1-9b88-85b7f9d27f25
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: ibm-spectrum-scale-csi-provisioner
subjects:
- kind: ServiceAccount
  name: ibm-spectrum-scale-csi-provisioner
  namespace: ibm-spectrum-scale-csi-driver
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 13:01:59
#
```

**Comment by gandhisanjayv** [comment]
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: "2020-03-20T00:14:38Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-attacher
  resourceVersion: "5387152"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterroles/ibm-spectrum-scale-csi-attacher
  uid: fad6440a-7832-486c-a390-b6f44e775c36
rules:
- apiGroups:
  - ""
  resources:
  - events
  verbs:
  - get
  - list
  - watch
  - update
- apiGroups:
  - ""
  resources:
  - persistentvolumes
  verbs:
  - get
  - list
  - watch
  - update
  - patch
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - storage.k8s.io
  resources:
  - volumeattachments
  verbs:
  - get
  - list
  - watch
  - update
  - patch
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: "2020-03-20T00:14:42Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-node
  resourceVersion: "5387165"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterroles/ibm-spectrum-scale-csi-node
  uid: 7c4e727d-d889-4954-952b-2903da263695
rules:
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
  - list
  - update
- apiGroups:
  - ""
  resources:
  - namespaces
  verbs:
  - get
  - list
- apiGroups:
  - ""
  resources:
  - persistentvolumes
  verbs:
  - get
  - list
  - watch
  - update
- apiGroups:
  - storage.k8s.io
  resources:
  - volumeattachments
  verbs:
  - get
  - list
  - watch
  - update
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: "2020-03-13T20:00:52Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-operator
  resourceVersion: "3820032"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterroles/ibm-spectrum-scale-csi-operator
  uid: 03f7b754-b35c-457d-81d5-b09904b801d2
rules:
- apiGroups:
  - ""
  resources:
  - pods
  - persistentvolumeclaims
  - services
  - endpoints
  - events
  - configmaps
  - secrets
  - secrets/status
  - services/finalizers
  - serviceaccounts
  verbs:
  - '*'
- apiGroups:
  - rbac.authorization.k8s.io
  resources:
  - clusterroles
  - clusterrolebindings
  verbs:
  - '*'
- apiGroups:
  - apps
  resources:
  - deployments
  - daemonsets
  - replicasets
  - statefulsets
  verbs:
  - '*'
- apiGroups:
  - monitoring.coreos.com
  resources:
  - servicemonitors
  verbs:
  - get
  - create
- apiGroups:
  - apps
  resourceNames:
  - ibm-spectrum-scale-csi-operator
  resources:
  - deployments/finalizers
  verbs:
  - update
- apiGroups:
  - apps
  resources:
  - replicasets
  verbs:
  - get
- apiGroups:
  - csi.ibm.com
  resources:
  - '*'
  verbs:
  - '*'
- apiGroups:
  - security.openshift.io
  resources:
  - securitycontextconstraints
  verbs:
  - '*'
- apiGroups:
  - storage.k8s.io
  resources:
  - volumeattachments
  - storageclasses
  verbs:
  - '*'
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: "2020-03-20T00:14:46Z"
  labels:
    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
    product: ibm-spectrum-scale-csi
    release: ibm-spectrum-scale-csi-operator
  name: ibm-spectrum-scale-csi-provisioner
  resourceVersion: "5387177"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterroles/ibm-spectrum-scale-csi-provisioner
  uid: a9f0c483-4f67-4a24-b351-84f9e7f0602a
rules:
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - get
  - list
- apiGroups:
  - ""
  resources:
  - persistentvolumes
  verbs:
  - get
  - list
  - watch
  - create
  - delete
- apiGroups:
  - ""
  resources:
  - persistentvolumeclaims
  verbs:
  - get
  - list
  - watch
  - update
- apiGroups:
  - storage.k8s.io
  resources:
  - storageclasses
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - storage.k8s.io
  resources:
  - csinodes
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - ""
  resources:
  - events
  verbs:
  - list
  - watch
  - create
  - update
  - patch
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 13:04:16
#
```

**Comment by gandhisanjayv** [comment]
```
kubectl get CSIScaleOperator -n ibm-spectrum-scale-csi-driver -o yaml
apiVersion: v1
items:
- apiVersion: csi.ibm.com/v1
  kind: CSIScaleOperator
  metadata:
    annotations:
      kubectl.kubernetes.io/last-applied-configuration: |
        {"apiVersion":"csi.ibm.com/v1","kind":"CSIScaleOperator","metadata":{"annotations":{},"labels":{"app.kubernetes.io/instance":"ibm-spectrum-scale-csi-operator","app.kubernetes.io/managed-by":"ibm-spectrum-scale-csi-operator","app.kubernetes.io/name":"ibm-spectrum-scale-csi-operator","release":"ibm-spectrum-scale-csi-operator"},"name":"ibm-spectrum-scale-csi","namespace":"ibm-spectrum-scale-csi-driver"},"spec":{"clusters":[{"id":"18295202154298832981","primary":{"primaryFs":"fs2","primaryFset":"csifset2","remoteCluster":"3071302939968702536","remoteFs":"fs2"},"restApi":[{"guiHost":"10.10.10.20"}],"secrets":"secret1","secureSslMode":false},{"id":"3071302939968702536","restApi":[{"guiHost":"10.10.10.110"}],"secrets":"secretr","secureSslMode":false}],"nodeMapping":[{"k8sNode":"archieprt01","spectrumscaleNode":"archieprt01-40"},{"k8sNode":"archieprt02","spectrumscaleNode":"archieprt02-40"},{"k8sNode":"archieprt03","spectrumscaleNode":"archieprt03-40"},{"k8sNode":"archieprt04","spectrumscaleNode":"archieprt04-40"},{"k8sNode":"archieprt05","spectrumscaleNode":"archieprt05-40"},{"k8sNode":"archieprt06","spectrumscaleNode":"archieprt06-40"},{"k8sNode":"archieprt07","spectrumscaleNode":"archieprt07-40"},{"k8sNode":"archieprt09","spectrumscaleNode":"archieprt09-40"}],"scaleHostpath":"/ibm/fs2"},"status":{}}
    creationTimestamp: "2020-03-20T00:14:18Z"
    finalizers:
    - finalizer.csiscaleoperators.csi.ibm.com
    generation: 1
    labels:
      app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
      app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
      app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
      release: ibm-spectrum-scale-csi-operator
    name: ibm-spectrum-scale-csi
    namespace: ibm-spectrum-scale-csi-driver
    resourceVersion: "5387462"
    selfLink: /apis/csi.ibm.com/v1/namespaces/ibm-spectrum-scale-csi-driver/csiscaleoperators/ibm-spectrum-scale-csi
    uid: d1888dbc-3eb4-421e-9852-80283ff6a299
  spec:
    clusters:
    - id: "18295202154298832981"
      primary:
        primaryFs: fs2
        primaryFset: csifset2
        remoteCluster: "3071302939968702536"
        remoteFs: fs2
      restApi:
      - guiHost: 10.10.10.20
      secrets: secret1
      secureSslMode: false
    - id: "3071302939968702536"
      restApi:
      - guiHost: 10.10.10.110
      secrets: secretr
      secureSslMode: false
    nodeMapping:
    - k8sNode: archieprt01
      spectrumscaleNode: archieprt01-40
    - k8sNode: archieprt02
      spectrumscaleNode: archieprt02-40
    - k8sNode: archieprt03
      spectrumscaleNode: archieprt03-40
    - k8sNode: archieprt04
      spectrumscaleNode: archieprt04-40
    - k8sNode: archieprt05
      spectrumscaleNode: archieprt05-40
    - k8sNode: archieprt06
      spectrumscaleNode: archieprt06-40
    - k8sNode: archieprt07
      spectrumscaleNode: archieprt07-40
    - k8sNode: archieprt09
      spectrumscaleNode: archieprt09-40
    scaleHostpath: /ibm/fs2
  status:
    conditions:
    - ansibleResult:
        changed: 13
        completion: 2020-03-20T00:15:54.460099
        failures: 0
        ok: 35
        skipped: 9
      lastTransitionTime: "2020-03-20T00:14:18Z"
      message: Awaiting next reconciliation
      reason: Successful
      status: "True"
      type: Running
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
[root@archieprt01 ibm-spectrum-scale-csi-logs_03-20-2020-12:30:22] 2020-03-20 13:07:22
#
```

**Comment by mew2057** [comment]
Fix has been merged to the 1.1.0  branch.

**Comment by mew2057** [comment]
The attacher cluster role was missing the csinodes resource rules. This  resulted in the operator being unable to query  the csinodes. The  PR fixes this permission.
